tests/featureTesting/compare.py

```diff
@@ -0,0 +1,211 @@
+"""
+This module compares different output of cli
+
+List of functions:
+
+1. compare: Compare all the outputs generated from multiple
+    cli instances
+
+2. load_files: Loads every file present in the input list into a
+    pandas dataframe
+
+3. remove_outliers: Remove rows whose mzs and rts are not equal
+
+4. remove_outliers_using_diff: Remove outliers in pandas dataframe
+    on the basis of mzs and rts difference
+
+5. remove_outliers_using_score: Remove outliers in pandas dataframe
+    on the basis of score and duplicates
+
+6. plot: Plot scatter plot for comparison single sample from two pandas
+        dataframes
+
+7. get_layout: Returns layout for plotly
+
+"""
+import os
```

Comment:
follow PEP 8 guidelines for imports. Link is [here](https://www.python.org/dev/peps/pep-0008/#imports)